UnitOfWork/Models/BaseEntity.cs

```diff
@@ -0,0 +1,8 @@
+﻿namespace UnitOfWork.Models;
+
+public class BaseEntity
+{
+    public int Id { get; set; }
```

Comment:
Recommend always initialize fields with `default`, in this case `public int Id { get; set; } = default!;`

Reply:
fixed